recipes/crowcpp-crow/all/conanfile.py

```diff
@@ -31,7 +34,11 @@ def build(self):
 
     def package(self):
         self.copy(pattern="LICENSE*", dst="licenses", src=self._source_subfolder)
-        self.copy("*.h", dst=os.path.join("include", "crow"), src=os.path.join(self._source_subfolder, "include"))
+        self.copy(
```

Comment:
Also, please don't "explode" the function calls on multiple lines here and like you did in the `source` method.

Reply:
What formatter is doing this?

Reply:
[Black](https://github.com/psf/black). It's PEP8 compliant, and what I always use. When arguments don't fit on the 90 char width, they are re-formatted. Conan even asks if you used a PEP 8 compliant formatter...

Reply:
Some time ago, we agreed upon allowing long lines.
See our .editorconfig:
https://github.com/conan-io/conan-center-index/blob/446750baa6564214217172f15815349448bc76fa/.editorconfig#L12

The question about pep8 in the issue template is mostly there to make people think about formatting.
Perhaps it's time to think about adding a `pyproject.toml` to this repo.

Reply:
@prince-chrismc @SpaceIm @uilianries 
I experimented a bit with the `pyproject.toml` below.
Save it as **pyproject.yoml** in the root folder of conan-center-index,
install `black` using `python -m pip install --user black`.
Save your current git checkout and run black on all sources:
`find . -iname conanfile.py -exec python -m black {} \;`
```toml
[tool.black]
line-length = 200
target-version = ['py27']
```
I think the changes are acceptable.
The main issue is for `options` dicts that don't end with a comma, 
black will collapse it into one line instead of one line per option.
But this should be addressed on recipe base.
`option = {"shared": [True, False], "fPIC": [True, False]}`
should become
`option = {"shared": [True, False], "fPIC": [True, False],}`